app/components/number-widget/component.js

```diff
@@ -7,6 +7,7 @@ export default Ember.Component.extend({
     settings : {
         fontSize: 3,
         fontColor: '#F44336',
+        noLink: false,
```

Comment:
I think this boolean would be easier to understand if it was `link: true` vs. a double negative